pyproject.toml

```diff
@@ -55,26 +56,29 @@ docs = [
   "sphinx-gallery",
 ]
 
+[project.scripts]
+pybamm-cookiecutter = "pybamm_cookiecutter.cli:pybamm_cookiecutter_cli"
+
+[project.entry-points."pipx.run"]
+pybamm-cookiecutter = "pybamm_cookiecutter.cli:pybamm_cookiecutter_cli"
+
 [project.urls]
 Homepage = "https://github.com/pybamm-team/pybamm-cookiecutter"
 "Bug Tracker" = "https://github.com/pybamm-team/pybamm-cookiecutter/issues"
 Discussions = "https://github.com/pybamm-team/pybamm-cookiecutter/discussions"
 Changelog = "https://github.com/pybamm-team/pybamm-cookiecutter/releases"
 
-[project.entry-points."parameter_sets"]
-Chen2020 = "pybamm_cookiecutter.parameters.input.Chen2020:get_parameter_values"
-
-[project.entry-points."models"]
-SPM = "pybamm_cookiecutter.models.input.SPM:SPM"
-BasicReservoir = "pybamm_cookiecutter.models.input.BasicReservoir:BasicReservoir"
-
 [tool.hatch]
 version.source = "vcs"
 build.hooks.vcs.version-file = "src/pybamm_cookiecutter/_version.py"
 envs.default.dependencies = [
   "pybamm",
 ]
 
+[tool.hatch.build.targets.wheel]
+only-include = ["src/pybamm_cookiecutter", "/template", "copier.yml"]
+sources = ["src"]
+
```

Comment:
Does this configuration include the LICENSE and the bundled licenses? Could you unpack both the sdist and wheel and verify? If it doesn't, we should include those files as well.

Reply:
They don't, will add them!